app/population/controllers/weekly.js

```diff
@@ -0,0 +1,24 @@
+function dashboard(req, res) {
+  const {
+    categoryName,
+    context,
+    dateRange,
+    datePagination,
+    resultsAsPopulationTables,
+    locationName,
+  } = req
+  const { period } = req.params
+
+  res.render('population/views/weekly', {
+    context,
+    pageTitle: 'dashboard::page_title',
+    datePagination,
+    resultsAsPopulationTables,
+    period,
+    dateRange,
+    location: locationName,
```

Comment:
These two are passed in to build the `Category : Location` heading at the top of the page.